apps/src/componentLibrary/simpleDropdown/simpleDropdown.module.scss

```diff
@@ -0,0 +1,238 @@
+@import "color";
+@import "@cdo/apps/componentLibrary/common/styles/mixins";
+
+// Dropdown common styles
+.dropdownContainer {
+  position: relative;
+  display: inline-block;
+  margin: 0;
+
+  .dropdownLabel {
+    display: block;
+  }
+
+  .dropdown {
+    display: inline-flex;
+    align-items: center;
+    align-self: stretch;
+    width: auto;
+    background-color: unset;
+    border-radius: 0.25rem;
+    border: 1px solid;
+    color: $light_black;
```

Comment:
Isn't  `color` set below by 'black' or 'white'?

Reply:
Yes it is, thanks for spotting!